2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten] MongoDB starting : pid=8 port=27017 dbpath=/data/db 64-bit host=b8901450b167
2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten] db version v3.6.3
2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten] git version: 9586e557d54ef70f9ca4b43c26892cd55257e1a5
2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.1.0g  2 Nov 2017
2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten] modules: none
2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten] build environment:
2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten]     distarch: x86_64
2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2019-06-22T19:54:56.410+0000 I CONTROL  [initandlisten] options: { net: { bindIp: "0.0.0.0", port: 27017 }, processManagement: { fork: true }, storage: { dbPath: "/data/db" }, systemLog: { destination: "file", path: "/data/logs" } }
2019-06-22T19:54:56.410+0000 W -        [initandlisten] Detected unclean shutdown - /data/db/mongod.lock is not empty.
2019-06-22T19:54:56.410+0000 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2019-06-22T19:54:56.411+0000 W STORAGE  [initandlisten] Recovering data from the last clean checkpoint.
2019-06-22T19:54:56.411+0000 I STORAGE  [initandlisten] 
2019-06-22T19:54:56.411+0000 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2019-06-22T19:54:56.411+0000 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2019-06-22T19:54:56.411+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7454M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2019-06-22T19:54:57.362+0000 I STORAGE  [initandlisten] WiredTiger message [1561233297:362047][8:0x7f4f779fef80], txn-recover: Main recovery loop: starting at 43/3456
2019-06-22T19:54:57.362+0000 I STORAGE  [initandlisten] WiredTiger message [1561233297:362619][8:0x7f4f779fef80], txn-recover: Recovering log 43 through 44
2019-06-22T19:54:57.486+0000 I STORAGE  [initandlisten] WiredTiger message [1561233297:486200][8:0x7f4f779fef80], file:collection-0--8446467013468135376.wt, txn-recover: Recovering log 44 through 44
2019-06-22T19:54:58.146+0000 I CONTROL  [initandlisten] 
2019-06-22T19:54:58.146+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2019-06-22T19:54:58.146+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2019-06-22T19:54:58.146+0000 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
2019-06-22T19:54:58.146+0000 I CONTROL  [initandlisten] 
2019-06-22T19:54:58.168+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2019-06-22T19:54:58.169+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
2019-06-22T19:54:58.505+0000 I NETWORK  [listener] connection accepted from 172.19.0.1:58482 #1 (1 connection now open)
